src/components/RewardWrapper/index.tsx

```diff
@@ -23,6 +24,10 @@ import { useCanClaimReward } from '@/hooks/useCanClaimReward'
 import { useGetUserPlayStreak } from '@/hooks/useGetUserPlayStreak'
 import { switchChain } from '@wagmi/core'
 
+function errorIsSwitchChainError(error: Error) {
```

Comment:
nitpick: you can use a type guard here

Reply:
Are you good with this now after the big :brain:  typescript changes you made :smile: ?